doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_pm_optimization.rst

```diff
@@ -64,6 +64,125 @@ The power management subsystem in a local domain is responsible for scheduling a
 The wake-up time scheduled in advance by the power management subsystem is combined with the advance time added by the software module.
 This approach ensures that the local domain and the software modules anticipating an event have sufficient time to fully restore before the event occurs, allowing the event to be handled without latency.
 
+Logical domains, power domains, and locality
+============================================
+
+The nRF54H20 SoC groups hardware into logical domains (software-visible functional groupings) and power domains (granular silicon regions that can be clock- or power-gated independently).
+
+The two domain types are defined as follows:
+
+Logical domain
+  Aggregates cores, memories, and peripherals behind a common isolation/security boundary (for example, application domain, radio domain, global domain).
+
+Power domain
+  A physical silicon region with its own retention/leakage characteristics and voltage/frequency constraints (for example, high-speed vs low-leakage subregions inside the application logical domain).
+
+A single logical domain can span multiple power domains.
+A logical domain frequently contains a low-leakage power domain (cheap retention, low frequency) and one or more high-speed power domains (higher leakage, higher attainable frequency).
+DVFS applies to high-speed domains.
+
+Locality principle
+------------------
+
+Power optimization is driven by keeping execution and data access local:
+
+* Prefer executing code from local RAM (TCM/local SRAM) within the active core's logical domain.
+* Minimize accesses to MRAM (global non-volatile memory) to avoid waking higher-power global domains and incurring cache refill energy.
```

Comment:
* Minimize accesses to MRAM (global non-volatile memory) to avoid waking **other** domains and incurring cache refill energy.